src/remix.ts

```diff
@@ -66,27 +93,8 @@ export const serve: ServeHandler = (nameOrInngest, fns, opts): unknown => {
         },
       };
     },
-    ({ body, status, headers }): Response => {
-      /**
-       * If `Response` isn't included in this environment, it's probably a Node
-       * env that isn't already polyfilling. In this case, we can polyfill it
-       * here to be safe.
-       */
-      let Res: typeof Response;
-
-      if (typeof Response === "undefined") {
-        // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-var-requires
-        Res = require("cross-fetch").Response;
-      } else {
-        // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
-        Res = Response;
-      }
-
-      return new Res(body, {
-        status,
-        headers,
-      });
-    }
+    createNewResponse,
+    createNewResponse
```

Comment:
why twice?

Reply:
Two separate definitions that happen to be the same in this case: https://github.com/inngest/inngest-js/blob/9aa99f4a26220a672dbec78ab89fc23404fc29ba/src/components/InngestCommHandler.ts#L364-L407